README.md

```diff
@@ -181,37 +181,37 @@ That way, your release code will contain no reference to LeakCanary other than t
 
 ### Icon and label
 
-`DisplayLeakActivity` comes with a default icon and label, which you can change by providing `R.drawable.__leak_canary_icon` and `R.string.__leak_canary_display_activity_label` in your app:
+`DisplayLeakActivity` comes with a default icon and label, which you can change by providing `R.drawable.leak_canary_icon` and `R.string.leak_canary_display_activity_label` in your app:
 
 ```
 res/
   drawable-hdpi/
-    __leak_canary_icon.png
```

Comment:
Release branches could be just fine right?
Something like `releases/1.4` containing updated versions of README, CHANGELOG, etc.